src/client/eppo-precomputed-client-with-bandits.spec.ts

```diff
@@ -102,6 +103,11 @@ describe('EppoPrecomputedClient Bandits E2E test', () => {
         const precomputedConfiguration = client.getBanditAction('banner_bandit_flag', 'nike');
         expect(precomputedConfiguration).toEqual({ action: null, variation: 'nike' });
       });
+
+      it('should return the assigned variation if a flag is not a bandit', () => {
+        const precomputedConfiguration = client.getBanditAction('not-a-bandit-flag', 'default');
+        expect(precomputedConfiguration).toEqual({ action: null, variation: 'control' });
```

Comment:
also consider a situation where a flag IS a bandit flag, but they got assigned a non-bandit variation.

Also we should be using mocks to make sure the proper event loggers are called or not

Reply:
> also consider a situation where a flag IS a bandit flag, but they got assigned a non-bandit variation.

From the UFC resopnse, wouldn't that be the same situation? The flag would be in `flags` but not in `bandits`